runtime/compiler/control/JITServerHelpers.cpp

```diff
@@ -120,14 +120,20 @@ JITServerHelpers::insertIntoOOSequenceEntryList(ClientSessionData *clientData, T
 void
 JITServerHelpers::printJITServerMsgStats(J9JITConfig *jitConfig)
    {
+   int totalMsgCount = 0;	   
    PORT_ACCESS_FROM_JITCONFIG(jitConfig);
    j9tty_printf(PORTLIB, "JITServer Message Type Statistics:\n");
    j9tty_printf(PORTLIB, "Type# #called TypeName\n");
    for (int i = 0; i < JITServer::MessageType_ARRAYSIZE; ++i)
       {
       if (JITServerHelpers::serverMsgTypeCount[i] > 0)
+         {
          j9tty_printf(PORTLIB, "#%04d %7u %s\n", i, JITServerHelpers::serverMsgTypeCount[i], JITServer::messageNames[i]);
+         totalMsgCount += JITServerHelpers::serverMsgTypeCount[i];
+         }
       }
+   j9tty_printf(PORTLIB, "Total number of messages:%d\tAverage number of messages per compilation:%f\n", totalMsgCount, totalMsgCount/float(JITServerHelpers::serverMsgTypeCount[0]));
```

Comment:
We need a check that `JITServerHelpers::serverMsgTypeCount[0]` is not 0. Otherwise the JVM will crash when doing division by 0